pom.xml

```diff
@@ -1,12 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
```

Comment:
I wonder if removing this POM will change versions of dependencies or something else.

Reply:
Right now 5.0.26 version of `public-pom` is used. It doesn't include any versions. `public-pom` itself is derived from `../base-pom:5.0.26`. Unfortunately, this POM isn't available on Artifactory. [The latest available version - 5.0.9](https://packages.atlassian.com/ui/repos/tree/PomView/maven-public-local%2Fcom%2Fatlassian%2Fpom%2Fbase-pom%2F5.0.9%2Fbase-pom-5.0.9.pom) - indeed includes a lot of configuration, but it is mostly related to standard Maven plugins. Changing their version may have also introduced some issues, so it's good idea to test everything. So far
* unit and integration tests pass: https://github.com/atlassian-labs/atlassian-slack-integration-server/actions/runs/1639209071 ;
* publishing a new release works: https://github.com/atlassian-labs/atlassian-slack-integration-server/actions/runs/1637911749 .

So, some versions of plugins might have changed, but it didn't introduce any errors in the essential workflows.